pkg/agent/api/delegatedidentity/v1/service_test.go

```diff
@@ -457,20 +476,63 @@ func TestFetchJWTSVIDs(t *testing.T) {
 			selectors:    []*types.Selector{{Type: "sa", Value: "foo"}},
 			audience:     []string{"AUDIENCE"},
 			identities: []cache.Identity{
-				identityFromX509SVID(x509SVID1),
+				identities[0],
+			},
+			jwtSVIDsResp: map[spiffeid.ID]*client.JWTSVID{
+				id1: {
+					Token:     jwtSVID1.Marshal(),
```

Comment:
NIT: since you are using marshal for the very same token multiple times, what do you think about moving that into a variable? and just use vars here?